bin/env_variables_all.txt

```diff
@@ -28,6 +38,7 @@ CCD_BEFTA_SOLICITOR_3_PWD=Pa55word11
 CCD_CASEWORKER_AUTOTEST_EMAIL=auto.test.cnp@gmail.com
 CCD_CASEWORKER_AUTOTEST_FE_PASSWORD=Pa55word11
 CCD_CASEWORKER_AUTOTEST_PASSWORD=Pa55word11
+CCD_DATA_STORE_URL=http://localhost:4452
 CCD_DATA_STORE_API_BASE_URL=http://localhost:4452
```

Comment:
Do we need this defined twice or can we just adjust the case-doc code that uses the 'new' value?

Reply:
Will raise a PR to fix in case doc code